eng/Version.Details.xml

```diff
@@ -299,6 +299,7 @@
     <Dependency Name="Microsoft.DotNet.Arcade.Sdk" Version="6.0.0-beta.21167.3">
       <Uri>https://github.com/dotnet/arcade</Uri>
       <Sha>0ca849f0b71866b007fedaaa938cee63f8d056a6</Sha>
+      <SourceBuild RepoName="arcade" ManagedOnly="true"/>
```

Comment:
Out of curiosity, what exactly does this metadata do?

Reply:
`SourceBuild` causes the source-build targets in the Arcade SDK to download
[intermediate nupkg]s from the upstream repo's official build, rather than using
prebuilt binaries to fulfill the dependencies. Note that `RepoName` is used to
calculate the ID of the [intermediate nupkg]: the `Dependency` `Name` is
ignored by source-build.

`ManagedOnly` determines whether a RID suffix is necessary on the [intermediate
nupkg] ID.

See https://github.com/dotnet/source-build/blob/main/Documentation/planning/arcade-powered-source-build/onboarding/local-onboarding.md for additional details.